contracts/utils/test/lib_address_array.ts

```diff
@@ -1,11 +1,4 @@
-import {
-    addressUtils,
-    chaiSetup,
-    expectContractCallFailedAsync,
-    provider,
-    txDefaults,
-    web3Wrapper,
-} from '@0x/contracts-test-utils';
+import { addressUtils, chaiSetup, provider, txDefaults, web3Wrapper } from '@0x/contracts-test-utils';
```

Comment:
You might consider using Lawrence's new `blockchainTests` system. To make this change, you would remove any code relating to chai, import `expect` from `contracts-test-utils`, and then change:

```
describe('...', () => {
    before(async () => {
        blockchain_lifecycle.startAsync();
    });
    after(async () => {
       blockchain_lifecycle.revertAsync();
    });
});
```

to

```
blockchainTests.resets('...', env => // This env variable will have provider, txDefaults, and web3Wrapper
});
```

Essentially, this change makes things a little more succinct, matches the current style of tests used by the protocol team, and allows the `BlockchainLifecycle` dependency to be removed.

Reply:
discussed on Slack. this is only in the `3.0` branch right now.